Edge10.CouchDb.Client/BaseCouchModel.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Edge10.CouchDb.Client
+{
+	/// <summary>
+	/// A base class for <see cref="ICouchModel"/> instances.
+	/// </summary>
+	/// <typeparam name="TConcrete">The type of the concrete.</typeparam>
+	public abstract class BaseCouchModel<TConcrete> : ICouchModel where TConcrete : ICouchModel
+	{
```

Comment:
I prefer `CouchModelBase` as a naming convention - better than you see all your `CouchModel*` classes sorted next to each other than all of your `Base*` classes!